src/routes.py

```diff
@@ -12,15 +15,59 @@ def home():
 	return render_template('index.html', endpoint=endpoint, client_id=client_id, scope=scope,
 		response_type=response_type, redirect_uri=redirect_uri)
 
+def decode_access_token(token):
+    try:
+        _, payload, _ = token.split('.')
+        payload_decoded = base64.urlsafe_b64decode(add_padding(payload)).decode('utf-8')
+        return json.loads(payload_decoded)
+    except Exception as e:
+        logging.error(f"Error decoding JWT: {e}", exc_info=True)
+        raise
+
+def add_padding(token_part):
+    """Adds the required padding to the base64 encoded string."""
+    return token_part + '=' * (4 - len(token_part) % 4)
+
+def get_cache(cache_key):
+     return print(cache.get(cache_key))
```

Comment:
Why are you returning the result of a print function, either just print the thing and don't return anything. Or return a value and use it.